### PR TITLE
LPS-172414 -  [Raylife AP] - Create generic component Show/Hide

### PR DESCRIPTION
**Pull request by eduardo-diniz**
Task: [LPS-171536](https://issues.liferay.com/browse/LPS-171536)

### CODE REVIEW

**Comment by eduardo-diniz** [comment]
ci:test:sf

**Comment by eduardo-diniz** [comment]
ci:test:sf

File: modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/policies/pages/components/PolicyDetail/index.tsx [hunk 1]
```diff
@@ -75,8 +85,45 @@ const PolicyDetail = ({
...
+		if (itemA.date > itemB.date) {
+			return -1;
+		}
+
+		if (itemA.date < itemB.date) {
+			return 1;
+		}
+
+		return 0;
+	};
```
Comment (joanagrego) [inline modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/policies/pages/components/PolicyDetail/index.tsx:112]: Please change it to a generic file.

**Review by joanagrego — CHANGES_REQUESTED** [review]
Good job! I added some suggestions :)

File: modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/policies/pages/components/PolicyDetail/index.tsx (hunk 1 above)
Comment (eduardo-diniz) [reply modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/policies/pages/components/PolicyDetail/index.tsx:112]: Done!

File: modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/policies/pages/components/PolicyDetail/index.scss [hunk 2]
```diff
@@ -58,4 +60,107 @@
...
+	}
+
+	.history-detail-border {
+		border-bottom-left-radius: 8px;
+		border-bottom-right-radius: 8px;
+		height: 100%;
+		position: relative;
+	}
+
+	.gsdc-panel::after {
```
[the remark this answers is not in the document]
Comment (eduardo-diniz) [reply modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/policies/pages/components/PolicyDetail/index.scss:79]: Changed for "dotted-line".

File: modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/policies/pages/components/PolicyDetail/index.scss [hunk 3]
```diff
@@ -58,4 +60,107 @@
 		position: absolute;
 		right: 0;
 	}
+
+	.layout-show-details {
+		background: #f2f6ff;
```
[the remark this answers is not in the document]
Comment (eduardo-diniz) [reply modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/policies/pages/components/PolicyDetail/index.scss:65]: Dialect don't have this exact color.

File: modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/policies/pages/components/PolicyDetail/index.scss [hunk 4]
```diff
@@ -58,4 +60,107 @@
...
+
+	.data-panel-hide {
+		margin-top: 13px;
+	}
+
+	.history-detail-border {
+		border-bottom-left-radius: 8px;
+		border-bottom-right-radius: 8px;
+		height: 100%;
+		position: relative;
```
[the remark this answers is not in the document]
Comment (eduardo-diniz) [reply modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/policies/pages/components/PolicyDetail/index.scss:76]: Done!

File: modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/policies/pages/components/PolicyDetail/index.scss [hunk 5]
```diff
@@ -58,4 +60,107 @@
...
+		}
+	}
+}
+
+@media screen and (max-width: 700px) {
+	.policy-detail-container {
+		background: none;
+		padding-top: 15px !important;
+	}
 }
```
[the remark this answers is not in the document]
Comment (eduardo-diniz) [reply modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/policies/pages/components/PolicyDetail/index.scss:166]: I removed some, but the remaining is necessary.

**Comment by eduardo-diniz** [comment]
ci:test:sf

**Review by joanagrego — APPROVED** [review]
Approved! Thank you @eduardo-diniz

**Comment by joanagrego** [comment]
ci:test/bundle

**Comment by eduardo-diniz** [comment]
ci:test:bundle

**Comment by nikki-pru** [comment]
ci:test:stable

**Comment by eduardo-diniz** [comment]
ci:test:sf

**Comment by eduardo-diniz** [comment]
ci:test:bundle

**Comment by styller1** [comment]
ci:test:site-initializer-raylife-ap

**Comment by eduardo-diniz** [comment]
ci:test:site-initializer-raylife-ap

**Comment by nikki-pru** [comment]
ci:test:stable

**Pull request closed without merging**